storage/database/src/main/java/de/danoeh/antennapod/storage/database/PodDBAdapter.java

```diff
@@ -1246,7 +1246,12 @@ public final Cursor getFeedStatisticsCursor(boolean includeMarkedAsPlayed, long
                         + "SUM(CASE WHEN " + TABLE_NAME_FEED_MEDIA + "." + KEY_DOWNLOAD_DATE + " > 0"
                                 + " THEN 1 ELSE 0 END) AS num_downloaded, "
                         + "SUM(CASE WHEN " + TABLE_NAME_FEED_MEDIA + "." + KEY_DOWNLOAD_DATE + " > 0"
-                                + " THEN " + TABLE_NAME_FEED_MEDIA + "." + KEY_SIZE + " ELSE 0 END) AS download_size"
+                                + " THEN " + TABLE_NAME_FEED_MEDIA + "." + KEY_SIZE + " ELSE 0 END) AS download_size, "
+                        + "CASE WHEN EXISTS ("
+                        + "  SELECT 1 FROM " + TABLE_NAME_FEED_ITEMS + " fi"
+                        + "  WHERE fi." + KEY_FEED + " = " + TABLE_NAME_FEEDS + "." + KEY_ID
+                        + "    AND fi." + KEY_READ + " != " + FeedItem.PLAYED
+                        + "    AND fi." + KEY_PUBDATE + " >= " + sixMonthsAgo + " ) THEN 1 ELSE 0 END AS has_recent_unplayed "
```

Comment:
With this subquery it will probably have to join the table with itself. Something like `SUM(CASE WHEN (<read> AND <recent>) THEN 1 ELSE 0 END)` should give the number of recent episodes without having to do an additional table scan